databasebuild.yaml

```diff
@@ -0,0 +1,19 @@
+#To use this you will need buildcomponents.yaml in your top level ci file. 
+
+build:
+  extends: .migrate
+  stage: build
+  services:
+    - $POSTGRES_VERSION
+  after_script:
+    - npm test
```

Comment:
If you are doing something node-specific, can you name the template (the filename) something with the word "node"? or "npm"?

Reply:
Changed how this is called so tell me if its still badly named.

Reply:
I don't think you pushed anything yet to change the name of this yet.